server/src/main/java/org/elasticsearch/index/reindex/ReindexRequest.java

```diff
@@ -262,6 +280,9 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         destination.writeTo(out);
         out.writeOptionalWriteable(remoteInfo);
+        if (out.getVersion().onOrAfter(Version.V_7_8_0)) {
```

Comment:
Same comment here about this having to be 8.0.0 until backport (or else the BWC tests will likely fail)

Reply:
Yep, you're right, changed it and will follow up after backport